README.md

```diff
@@ -121,3 +121,26 @@ are study related:  `["info", "telemetry", "endStudy"]`.
 5. Data is sent to Telemetry.
 7. The study ends and the addon is uninstalled.
 
+## Simulations
+To check that the addon is working properly, some simulations should be performed. For that:
```

Comment:
nit: "To validate using the simulator, the addon can be set to work in *simulation* mode:"

Reply:
Done